fdbserver/VersionedBTree.actor.cpp

```diff
@@ -954,6 +958,19 @@ class FIFOQueue {
 				if (c.pageID == invalidLogicalPageID || c.pageID == c.endPageID) {
 					debug_printf("FIFOQueue::Cursor(%s) peekAllExt Queue exhausted\n", c.toString().c_str());
 					res.send(results);
+
+					// Since we have reached the end of the queue, verify that the number of entries read matches
+					// the queue metadata. If it does, send end_of_stream() to mark completion, else throw an error
+					entriesRead += results.size();
+					if (entriesRead != self->numEntries) {
+						Error e = internal_error(); // TODO:  Something better?
+						TraceEvent(SevError, "FIFOQueueNumEntriesMisMatch")
```

Comment:
The trace event name should start with `Redwood`.

Reply:
All of them or just this one? I see a mix of trace event names in this file.

Reply:
Good point!  All of them.  Let's change "DWALPager*" to "Redwood*", that should cover everything.

Reply:
Done